web/src/app/dialogs/FormDialog.js

```diff
@@ -64,6 +64,9 @@ export default class FormDialog extends React.PureComponent {
     onClose: p.func,
     onSubmit: p.func,
 
+    // disables form submit button
+    submitDisabled: p.bool,
```

Comment:
Do we have a use-case where we need to disable submitting? Everywhere else we allow submit and then display errors.

If they shouldn't submit, then when clicking display an error message (e.g. "You must click Send first before you can submit a code") so the user isn't left wondering why they can't submit the form.

In the case of CM though, it's possible the user already has a code (from having the form open previously) that they want to enter without sending again.

Reply:
It was a suggestion by @arurao. I'm open to discussion -- I don't really have a preference.

Reply:
That's correct. I did make this suggestion, but since we changed our design a little after that - sending the code on 1-clicking of the icon or Reactivate button, it seems that we could actually no longer need it. Meaning we could just remove the disabling Submit button entirely now. Since clicking on one of those two will send the code and then open the dialog, in which case when the dialog is open, there would already have had a code sent. So Submit can always be enabled.